pyiron/atomistics/job/atomistic.py

```diff
@@ -833,6 +833,14 @@ def energy_tot(self):
     def forces(self):
         return self._job["output/generic/forces"]
 
+    @property
+    def f_max(self):
```

Comment:
Maybe `force_max` is a better name? But this is fine too

Reply:
Yeah I also wondered whether `f_max` or `force_max`. I just thought for auto complete it might be easier to have something that doesn't start with `force`, but it's true that according to our general strategy it's probably better to write it more explicitly.

Reply:
But I guess before changing it back and forth, I'll wait for @jan-janssen and @liamhuber

Reply:
I prefer `force_max` as well, because the user wants the forces and then from the forces the maximum. I could also imagine having something like `force.max` in future and then `force_max` is closer to that already.

Reply:
That's a very good point. I guess the biggest problem is that numpy doesn't have something like `forces.norm(axis=-1).max(axis=-1)`, which would be acceptable by me.